targets/zephyr/src/main-zephyr.c

```diff
@@ -159,6 +172,13 @@ void main (void)
 {
   printf ("Jerry Compilation " __DATE__ " " __TIME__ "\n");
   jerry_init (JERRY_INIT_EMPTY);
+  jerry_value_t global_obj_val = jerry_get_global_object ();
+
+  jerry_value_t print_func_name_val = jerry_create_string ((jerry_char_t *) "print");
+  print_function = jerry_get_property (global_obj_val, print_func_name_val);
```

Comment:
The original idea was along the lines I mentioned previously: "We know that call is not expected to fail, and this is an embedded port, so it's not exactly clear how to deal with failure". But I guess you're right, something may happen with a later refactor (e.g. "print" replaced with "console.log"), so I'll try to add some no-nonsense error handling for that case.